handbook/technical-overview/api-design-guide/rest-request.md

```diff
@@ -0,0 +1,22 @@
+# REST Request
+
+## Query parameters
+
+### Working with sensitive data
```

Comment:
We also need to document how to handle sensitive resource ids, like /users/1234567890.

Might be worth mentioning that this is mainly a concern when dealing with national ids, and APIs should generally prefer non-sensitive IDs, like guids.